base/latex_symbols.jl

```diff
@@ -84,6 +84,10 @@ const latex_symbols = Dict(
     "\\backpprime" => "‶",
     "\\backppprime" => "‷",
     "\\xor" => "⊻",
+    "\\iff" => "⟺",
+    "\\implies" => "⟹",
+    "\\impliedby" => "⟸",
+    "\\to" => "→",
```

Comment:
No, in LaTeX with amsmath `\to` gives the same result as `\rightarrow`, not `\longrightarrow`. Also, we want `\to` to have the same length as `\mapsto` (not `\longmapsto`).

It would be nice to have `\longto` as well, but that is not standard :)